pypeit/biasframe.py

```diff
@@ -82,15 +74,14 @@ def build_image(self, overwrite=False, trim=True):
         Grab the bias files (as needed) and then
          process the input bias frames with ProcessImages.process()
           Avoid bias subtraction
-          Avoid trim
 
-        Parameters
-        ----------
-        overwrite : bool, optional
+        Args:
+            overwrite: (:obj: `bool`, optional):
+                Regenerate the stack image
+            trim (bool, optional): If True, trim the image
 
-        Returns
-        -------
-        stack : ndarray
+        Returns:
+            ndarray: :attr:`stack` Combined, processed image
```

Comment:
We may want to define a pypeit format for things like this, where we specify the name of the variable being returned.  Generally, you don't include this.  I.e., the user can name the returned variable whatever they want.

Reply:
I only called it out because it is an internal attribute.
Not really needed.